tb/cocotb/soc/test_pll_apb_regs.py

```
@@ -0,0 +1,278 @@
+      Write all-ones to STATUS; confirm STATUS unchanged (RO).
+
+  test_pll_apb_regs_decode_and_enable
+      Write+read CONTROL; confirm pslverr=0 (OKAY), pll_enable_o toggles.
+"""
+
+import sys
+from pathlib import Path
+
+import cocotb
+from cocotb.clock import Clock
+from cocotb.triggers import RisingEdge, ClockCycles
```

Flake8 (7.3.0)

[error] 'cocotb.triggers.RisingEdge' imported but unused

(F401)